SECURITY.md

```diff
@@ -4,15 +4,11 @@
 
 | Version   | Supported          |
 | -------   | ------------------ |
-| 1.14.x    | :white_check_mark: |
-| <= 1.13.x | :x:                |
+| 1.16.x    | :white_check_mark: |
+| 1.15.x    | :white_check_mark: |
```

Comment:
I'm planning to release fluentd v1.15.4 & td-agent v4.4.3 with some backport patches, so 1.15.x is still supported.

Reply:
Can we add documentation for supporting strategies/periods for multiple versions?
To stabilize maintenance process is needed, I guess.

e.g.)
When the new version v1.16 is released, the previous series should be supported after 6 months later or something like that.

Reply:
Yes, it should be clarified later. Anyway, main scope of this PR is updating contact address.

Reply:
We'll clarify it after LTS release is on rail.